kratos/processes/fast_transfer_between_model_parts_process.cpp

```diff
@@ -161,7 +167,7 @@ void FastTransferBetweenModelPartsProcess::TransferWithFlags()
 
         if (num_constraints != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::CONSTRAINTS || mEntity == EntityTransfered::NODESANDCONSTRAINTS)) {
             const auto it_const_begin = mrOriginModelPart.MasterSlaveConstraintsBegin();
-            #pragma omp for
+            #pragma omp for schedule(guided, 512)
```

Comment:
I'd leave it as it is considering that (eventually) we'll upgrade this to use the parallel utils, which have no scheduling support.